docs/sw-update-conf.json

```diff
@@ -14,110 +14,81 @@
     },
     "restart": "environment"
   },
-  "octoprint": {
-    "pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-    "prerelease": false,
-    "develop":{
-      "prerelease": true
-    },
-    "beamos_date": {
-      "2021-06-11": {
+  "modules": {
+    "octoprint": {
+      "type": "github_release",
+      "develop": {
+        "type": "github_commit"
+      },
+      "beta": {
+        "prerelease_channel": "mrbeam2-{tier}",
+        "prerelease": true
+      },
+      "alpha": {
+        "prerelease_channel": "mrbeam2-{tier}",
         "prerelease": true
       }
-    }
-  },
-  "mrbeam": {
-    "name": " MrBeam Plugin",
-    "repo": "MrBeamPlugin",
-    "restart": "octoprint",
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeam": {
+      "name": " MrBeam Plugin",
+      "repo": "MrBeamPlugin",
+      "restart": "octoprint",
+      "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
+      "submodules": {
```

Comment:
Do you mean "dependencies" ?

Reply:
And what is the goal with grouping them under the mrb plugin? they are inherently indepedent and installed in a different venv

Reply:
hmm, yeah this can be a better name

Reply:
these will be installed via the update_script and not by octoprint itselfe

Reply:
so only this "submodule" category gets passed to an installer script?

Reply:
that is the plan

Reply:
are we changing the name to dependencies or leaving submodules @Josef-MrBeam? 🤔

Reply:
ah sorry I see it was already changed 🤦